packages/datadog-esbuild/index.js

```diff
@@ -2,73 +2,111 @@
 
 /* eslint-disable no-console */
 
-const NAMESPACE = 'datadog'
-
-const instrumented = Object.keys(require('../datadog-instrumentations/src/helpers/hooks.js'))
-const rawBuiltins = require('module').builtinModules
+const path = require('path')
+const fs = require('fs')
+const instrumentations = require('../datadog-instrumentations/src/helpers/instrumentations.js')
 
 warnIfUnsupported()
 
+{
+  // run each instrumentation so that `instrumentations` gets populated
+  const INS_PATH = path.join(__dirname, '../datadog-instrumentations/src')
```

Comment:
It would be simpler to just require instead of manipulating the filesystem directly. It would also avoid issues if the structure of the folder changes or if unrelated files are added.

Reply:
Hmm... I suppose I can require `packages/datadog-instrumentations/src/helpers/hooks.js` then loop through and execute each object value.

Reply:
This change has been made.